bc/channel/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+from bc.core.models import AbstractDateTimeModel
+from bc.users.models import User
+
+from .utils.masto import masto_regex
+
+
+class Channel(AbstractDateTimeModel):
+    """
+    A "Channel" is a particular account on a service, which is used by
+    BCB to broadcast or to issue commands to BCB.
+    """
+
+    TWITTER = "t"
+    MASTODON = "m"
+    CHANNELS = (
+        (TWITTER, "Twitter"),
+        (MASTODON, "Mastodon"),
+    )
+    service = models.CharField(
+        help_text="Name of the service",
+        max_length=100,
+        choices=CHANNELS,
+    )
```

Comment:
Let's make this a [SmallIntegerField](https://docs.djangoproject.com/en/3.0/ref/models/fields/#smallintegerfield)

Reply:
Alright!

Reply:
Thanks. I couldn't even decide why I cared, but I do? We use charfields for this in some parts of CL and they just seem wrong, though I liked them at the time. I don't know exactly why. I guess I should research it, but for now it's a gut feeling that char fields are wrong for this.